File: tms-angular2-master/src/app/core/models/ChildcareLeave.ts
export class ChildcareLeave {
    constructor(id: any, startdate: any, enddate: any, isLateComing: any, isEarlyLeaving: any, time: any) {
        this.Id = id;
        this.StartDate = startdate;
        this.EndDate = enddate;
        this.IsLateComing = isLateComing;
        this.IsEarlyLeaving = isEarlyLeaving;
        this.Time = time;
    }
    public Id: any
    public StartDate: any;
    public EndDate: any;
    public IsLateComing: any;
    public IsEarlyLeaving: any;
    public Time: any;
}

File: tms-webapi-master/TMS.Model/Models/ChildcareLeave.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TMS.Common.Constants;

namespace TMS.Model.Models
{
    [Table(CommonConstants.ChildcareLeave)]
    public class ChildcareLeave
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { set; get; }
        public string UserId { set; get; }
        public DateTime StartDate { set; get; }
        public DateTime EndDate { set; get; }
        public bool IsLateComing { set; get; }
        public bool IsEarlyLeaving { set; get; }
        public float Time { set; get; }
    }
}
